docs/en/cli.md

```diff
@@ -0,0 +1,475 @@
+  t1 21 0 172.24.4.55:9971 leader yes 1 1 301.000 9.353 K
+172.24.4.55:6531 demo_db> showtable auto_VCeOIIKA
+   name tid pid endpoint role is_alive offset record_cnt memused diskused
+-------------------------------------------------- -------------------------------------------------- --
+  auto_VCeOIIKA 25 0 172.24.4.55:9971 leader yes 8 4 498.000 9.128 K
+```
+
+#### showtablet
+
+View tablet information (if the serverName and automatic local ip function are used, the endpoint is serverName, and the real_endpoint is "-")
+
+```
+> showtablet
+  endpoint real_endpoint state age
+-------------------------------------------------- ------------
+  6534708411798331392 172.17.0.12:9531 kTabletHealthy 1d
+  6534708415948603392 172.17.0.13:9532 kTabletHealthy 1d
+  6534708420092481536 172.17.0.14:9533 kTabletHealthy 14h
+```
+
+#### addreplica
+
+Add replica
+
+Command format: `addreplica table_name pid_group endpoint`
+
+* table\_name table name
+* pid\_group shard id collection. There can be the following situations
+    * a single shard
+    * Multiple shards, separated by commas. Such as 1,3,5
+    * Shard interval, the interval is a closed interval. For example, 1-5 means shard 1, 2, 3, 4, 5
+* endpoint The endpoint of the node to be added as a replica
+
+```
+> showtable test1
+  name tid pid endpoint role ttl is_alive compress_type offset record_cnt memused
+-------------------------------------------------- -------------------------------------------------- --------
+  test1 13 0 172.27.128.31:8541 leader 0min yes kNoCompress 0 0 0.000
+  test1 13 1 172.27.128.31:8541 leader 0min yes kNoCompress 0 0 0.000
+  test1 13 2 172.27.128.31:8541 leader 0min yes kNoCompress 0 0 0.000
+  test1 13 3 172.27.128.31:8541 leader 0min yes kNoCompress 0 0 0.000
+> addreplica test1 0 172.27.128.33:8541
+AddReplica ok
+> showtable test1
+  name tid pid endpoint role ttl is_alive compress_type offset record_cnt memused
+-------------------------------------------------- -------------------------------------------------- --------
+  test1 13 0 172.27.128.31:8541 leader 0min yes kNoCompress 0 0 0.000
+  test1 13 0 172.27.128.33:8541 follower 0min yes kNoCompress 0 0 0.000
+  test1 13 1 172.27.128.31:8541 leader 0min yes kNoCompress 0 0 0.000
+  test1 13 2 172.27.128.31:8541 leader 0min yes kNoCompress 0 0 0.000
+  test1 13 3 172.27.128.31:8541 leader 0min yes kNoCompress 0 0 0.000
+> addreplica test1 1,2,3 172.27.128.33:8541
+AddReplica ok
+> addreplica test1 1-3 172.27.128.33:8541
+AddReplica ok
+```
+
+#### delreplica
+
+Delete replica
+
+Command format: `delreplica table_name pid_group endpoint`
+
+* table\_name table name
+* pid\_group shard id collection. There can be the following situations
+    * a single shard
+    * Multiple shards, separated by commas. Such as 1,3,5
+    * Shard interval, the interval is a closed interval. For example, 1-5 means shard 1, 2, 3, 4, 5
+* endpoint endpoint to delete replicas
```

Comment:
please replace those description as the same as the that of addreplica

Reply:
Ok, thank you.